docker/.env.docker.prod

```diff
@@ -0,0 +1,16 @@
+########## .env.docker ########################################################
+#
+# See packages/venia-concept/.env.dist file for full option details
+#
+###############################################################################
+
+NODE_ENV=production
+PORT=8080
+PWA_STUDIO_HOST=localhost
+# magento enterprise edition - in production mode
+MAGENTO_BACKEND_URL=https://m231-pwa-ent-1.testsonfire.com/
+MAGENTO_BUILDPACK_PROVIDE_SECURE_HOST=0
+UPWARD_JS_UPWARD_PATH=venia-upward.yml
+UPWARD_JS_BIND_LOCAL=1
+UPWARD_JS_LOG_URL=1
+BRAINTREE_TOKEN=sandbox_8yrzsvtm_s2bg8fs563crhqzk
```

Comment:
token should be pulled in from env vars rather than plain text and other files as well.